### DISCUSSION TITLE
Regolith 3 - Naming Sway workspaces in Xresources

### DISCUSSION THREAD

**Discussion by eljejer** [regolith-linux/regolith-desktop#904] [Q&A]
I'm trying to give my work spaces a name by adding wm.workspace.XX.name in Xresources:

```
regolith.wallpaper.file: /usr/share/backgrounds/WP_20150405_001.jpg
wm.program.lock: /home/eljejer/scripts/myi3lock.sh
wm.bar.position: top
wm.gaps.inner.size: 0
wm.program.files: /usr/bin/pcmanfm-qt
wm.binding.files: e
wm.workspace.01.name: terminal
wm.workspace.02.name: browser
wm.workspace.03.name: files
wm.workspace.04.name: mail
wm.workspace.05.name: joplin
wm.workspace.05.name: misc
!regolith.look.path: /usr/share/regolith-look/lascaille
regolith.look.path: /usr/share/regolith-look/dracula
```

After reload config  (Super+Shift+c) & refresh  session (Super+Shift+r) in a running session, this results in Super+number not navigating to the requested workspace. 
When starting a new session (log out and log in again) these entries in Xresources brings me to the desktop with 1 workspace 'terminal' and not being able to create wordspace 2, 3, and so on

**Comment by JakobRieckh** [comment]
Could it be that you named workspace 5 twice?
